.github/workflows/lambda-functions.yml

```diff
@@ -193,30 +163,8 @@ jobs:
           zip -ugj nightly_stats_bigquery_upload_lambda nightly_stats_bigquery_upload_lambda.py
           aws lambda update-function-code --function-name project-${{ inputs.environment }}-nightly-stats-bigquery-upload-lambda --zip-file fileb://nightly_stats_bigquery_upload_lambda.zip
 
-      - name: Package and deploy two-way sms lambda function
-        if: ${{ (inputs.lambdaName == 'TwoWaySMS') || (inputs.lambdaName == 'All') }}
+      - name: Package and deploy vetext incoming forwarder lambda
+        if: ${{ (inputs.lambdaName == 'VetTextIncomingForwarder') || (inputs.lambdaName == 'All') }}
         run: |
-          cd two_way_sms/
-          python3 -m venv venv_two_way_sms
-          source venv_two_way_sms/bin/activate
-          pip install -r requirements-lambda.txt
-          deactivate
-          cd venv_two_way_sms/lib/python3.8/site-packages
-          zip -r9 ../../../../two_way_sms.zip .
-          cd ../../../../
-          zip -ugj two_way_sms two_way_sms_v2.py
-          aws lambda update-function-code --function-name project-${{ inputs.environment }}-notify-incoming-sms-lambda --zip-file fileb://two_way_sms.zip
-
-      - name: Package and deploy nightly billing stats upload lambda function
-        if: ${{ (inputs.lambdaName == 'NightBillingBQUpload') || (inputs.lambdaName == 'All') }}
```

Comment:
Are you intentionally removing the deployment of the `nightly_billing_stats_lambda`?

Reply:
This is not intentional.  Thank you for the backup.

Reply:
restored